src/CMakeLists.txt

```diff
@@ -175,6 +178,15 @@ if(LIBRW_PLATFORM_GL3)
             )
         endif()
     endif()
+
+    if(NINTENDO_SWITCH)
+        target_link_libraries(librw
+            PUBLIC
+                EGL
+                glapi
+                drm_nouveau
```

Comment:
Can we assume these to be exist?
No need for a call to `find_library`?

Reply:
They're always present on the devkitPro Docker image, and can be trivially installed with devkitPro's package manager:
`(dpk-)pacman -S switch-mesa switch-glfw switch-drm_nouveau` (dpk- prefix on Unix systems, not needed on windows). devkitPro's toolchain file makes sure the right include/lib directories are set as well.

A find_library to make sure the libraries are actually there wouldn't hurt I guess.